docs/local-development.md

```diff
@@ -56,6 +56,21 @@ If not using macOS:
 
     You should now be able to go to open up your favorite browser, go to `localhost:3000` and see your local development environment running.
 
+### Running a sample partner application
+
+Typically, a person who uses Login.gov will arrive from a partner application, and their experience
+on Login.gov will be customized to incorporate the name and logo of the partner. They will also be
+asked to consent to share their information with the partner before being sent back.
+
+To simulate a true end-to-end user experience, you can use one of the sample partner applications,
+which are configured by default to work with your local IdP instance.
+
+- OIDC: https://github.com/18F/identity-oidc-sinatra
+- SAML: https://github.com/18F/identity-saml-sinatra
+
+When running a sample application, you can visit either http://localhost:9292/ (OIDC) or
+http://localhost:4567/ (SAML) and click "Sign in" to begin the sign-in process using the IdP.
+
```

Comment:
Does the `https://localhost:3000/test/saml/login` [tool/controller ](https://github.com/18F/identity-idp/pull/9506#discussion_r1385453209)fit into these docs as well? 

I'm not having any luck with `https://github.com/18F/identity-oidc-sinatra` because `make setup` requires ruby 2.7.1 and getting that installed is hard (from my experience last time).

Reply:
> because `make setup` requires ruby 2.7.1 and getting that installed is hard

I think we could update it if that helps!

Reply:
I actually was completely unaware of `/test/saml/login` until now, which I mentioned at https://github.com/18F/identity-idp/pull/9506#discussion_r1385466553 . Could be an interesting shortcut to mention here.

Reply:
Updating ruby would help me get that repo working for sure. On the other hand, I think I'm going to use the `/test/saml/login` method instead, so I'm not sure that's worth it?

Reply:
https://github.com/18F/identity-oidc-sinatra/blob/main/.ruby-version has `3.0.4`?

Reply:
This was my mistake. Thanks for catching that. `make setup` in that repo now works for me after I did `chruby 3.1.4`. My system version was `2.6.X` and that was the source of the problem (I should enable auto-switching based on .ruby-version).

Reply:
As noted in the other comment thread, I update the instructions in 1decdb7e3 to include a mention of the SAML test controller